drivers/pcd8544/pcd8544.c

```diff
@@ -30,7 +30,7 @@
 
 #define ASCII_MIN           0x20    /**< start of ASCII table */
 #define ASCII_MAX           0x7e    /**< end of ASCII table */
-#define CHAR_WIDTH          (6U)    /**< pixel width of a single character */
+#define CHAR_WIDTH          (6)    /**< pixel width of a single character */
```

Comment:
I don't think this change is needed, the previous version is more valid to me.

Reply:
Well the `(6U)` triggers several comparison warnings of different types. IMHO the `U` should be used when numbers are really high (maybe 16bit wide) to avoid casting errors on 8 bit platforms.

Thus in this case I don't see why it needs to be `(6U)`.

Reply:
I am with @aabadie here: I'd prefer leaving the `U`. IMHO this should not only used with high numbers, but it should be used to mark any unsigned value... So the preferable fix here would be to actually fix the code that uses this define (there are only 2 lines effected...)

Reply:
I agree, just a remark: can a width be negative?

Reply:
nope

Reply:
Why to make it unsigned then?

Reply:
If there is a problem with the sign when using, it should be modified when using it not here.

Reply:
Ok will change...